docker/app/Dockerfile

```diff
@@ -35,11 +40,26 @@ RUN set -xe \
     && apk add --no-cache --virtual .build-deps ${RUNTIME_DEPS} \
     # workaround for rabbitmq linking issue
     && ln -s /usr/lib /usr/local/lib64 \
+    # Build openssl
+    && mkdir -p /usr/local/src && cd $_ \
+        && wget "https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz" -O "openssl-${OPENSSL_VERSION}.tar.gz" \
+        && echo "$OPENSSL_SHA256" "openssl-${OPENSSL_VERSION}.tar.gz" | sha256sum -c - \
+        && tar -zxvf "openssl-${OPENSSL_VERSION}.tar.gz" \
+        && cd "openssl-${OPENSSL_VERSION}" \
+        && ./config no-async shared --prefix=/usr/local/ssl --openssldir=/usr/local/ssl -Wl,-rpath,/usr/local/ssl/lib \
+        && make && make install \
+        && cp /usr/local/ssl/bin/openssl /usr/bin/openssl \
+        && rm -rf "/usr/local/src/openssl-${OPENSSL_VERSION}.tar.gz" "/usr/local/src/openssl-${OPENSSL_VERSION}"
```

Comment:
This is required?

Reply:
for php:7.2.23 it isn't required. it was already removed (commented out)